store/src/java/com/zimbra/soap/SoapEngine.java

```diff
@@ -345,6 +346,13 @@ private Element dispatch(String path, Element envelope, Map<String, Object> cont
             if (doc.getName().equals("AuthRequest")) {
                 // this is a Auth request, no CSRF validation happens
                 doCsrfCheck = false;
+                try {
```

Comment:
You'll need to resolve this conflict with https://github.com/Zimbra/zm-mailbox/pull/463